packages/shared/lib/core/wallet/actions/claimActivity.ts

```diff
@@ -1,3 +1,5 @@
+import { syncBalance } from '@core/account/actions/syncBalance'
+import { selectedAccountId } from '@core/account/stores/selected-account-id.store'
 import { selectedAccount } from '@core/account/stores/selected-account.store'
```

Comment:
These should be able to be imported with in one statement; check to make sure the barrel files are exporting them.